src/ORM/Association/HasMany.php

```diff
@@ -441,6 +441,7 @@ public function replace(EntityInterface $sourceEntity, array $targetEntities, ar
         $ok = ($result instanceof EntityInterface);
 
         if ($ok) {
+            // phpcs:ignore SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable
             $sourceEntity = $result;
```

Comment:
Why can't we remove this?

Reply:
I think this is international so that the caller has access to the modified entity since the method retuns a boolean instead of the updated entity.

Reply:
Yes